exploration/nn_train.py

```diff
@@ -0,0 +1,103 @@
+'''
+This module is created to hold the neural network training function
+that is necessary to run weights and biases hyperparameter sweeps.
+https://github.com/wandb/client/issues/956
+per the above issue.
+
+This file is currently deprecated in favor of sweep_train.py
```

Comment:
Is this file still needed for this to work correctly?

(i tested this pull request by running sweep_train.py, which worked flawlessly)

Reply:
safe to remove in future pull request